### ISSUE TITLE
`thunder-ec2` launch fails due to git error.

### ISSUE THREAD

**Issue by bainzo** [thunder-project/thunder#246]
I'm trying to get Thunder going on EC2 by invoking `thunder-ec2` using v0.6.0 installed using PyPI.

During the Thunder installation stage (after the cluster is 'ssh-ready') the install fails with git complaining that the pathspec 'v0.6.0' is invalid.

I think this is because no commit is tagged 'v0.6.0' in the repository. Using the option `--thunder-version='0.5.1'` changes the error to one unrelated to git.

Here is the traceback:
```Python-traceback
Traceback (most recent call last):
  File "/home/user0/anaconda/envs/thunder-aws/lib/python2.7/site-packages/thunder/utils/ec2.py", line 541, in <module>
    install_thunder(master, opts)
  File "/home/user0/anaconda/envs/thunder-aws/lib/python2.7/site-packages/thunder/utils/ec2.py", line 206, in install_thunder
    ssh(master, opts, "cd thunder && git checkout %s" % tagOrHash)
  File "/home/user0/anaconda/envs/thunder-aws/lib/python2.7/site-packages/thunder/utils/ec2.py", line 300, in ssh
    raise Exception(stdout)
Exception: error: pathspec 'v0.6.0' did not match any file(s) known to git.
```

**Comment by freeman-lab** [comment]
Thanks for flagging @bainzo ! This has been fixed by tagging `branch-0.6` as `v0.6.0` on github. I just confirmed a successful cluster launch with a fresh pip installation.

**Comment by bainzo** [comment]
Thanks!
